lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -44,6 +44,54 @@ bool firrtl::isDuplexValue(Value val) {
       .Default([](auto) { return false; });
 }
 
+flow::Flow firrtl::foldFlow(Value val, flow::Flow acc) {
```

Comment:
Plz add a commend describing what this is.  Also, plz expand the name `acc` it isn't obvious to me.

Reply:
Will do. 

What's the guidance on documenting the declaration vs. documenting the implementation? For now, I'll follow the existing way the code is and document the declaration in the header.

Reply:
Yeah, can change `acc` to `accumulatedFlow` or something. I think `acc` is some haskell-ism I picked up for the name of the accumulated value in a fold.